src/shared/components/person/inbox.tsx

```diff
@@ -795,7 +795,7 @@ export class Inbox extends Component<any, InboxState> {
         limit,
       }),
     });
-    UnreadCounterService.Instance.update();
+    UnreadCounterService.Instance.updatePrivateMessages();
```

Comment:
Why only update private messages here?

Reply:
Only "private messages" (the counts of which are returned by the UnreadCounts API call) are displayed in the inbox so I can be certain that the function wasn't called because for instance a report was dismissed. Might be a bad name but the "updatePrivateMessages" function will update the counts for PMs, mentions and replies for the number next to the inbox button.

Would "updateInboxCounts" (doing the same thing) be better?

Reply:
Yes please change that name, private messages are only one of the three types of unreads for normal users.

Maybe `updatePersonUnreads`